lib/light-service/organizer/with_reducer.rb

```diff
@@ -25,7 +25,7 @@ def reduce(*actions)
         raise "No action(s) were provided" if actions.empty?
         actions.flatten!
 
-        actions.reduce(context) do |current_context, action|
+        actions.each_with_object(context) do |action, current_context|
```

Comment:
By using `each_with_object`, you can remove the `result` from line 38. That's one of the benefits of using `each_with_object`.
I tested one of our apps with that change, and it passed all the tests.

Reply:
good catch thanks